experimenter/tests/integration/nimbus/utils/helpers.py

```diff
@@ -160,7 +160,9 @@ def load_experiment_data(slug):
     )
 
 
-def create_basic_experiment(name, app, targeting, languages=[]):
+def create_basic_experiment(name, app, targeting, languages=None):
+    if languages is None:
+        languages = []
```

Comment:
Oh this is a really cool one, this is the problem in Python where you put a collection in a default arg of a function and it accidentally becomes a singleton which bleeds state between invocations @jrbenny35 That's smart.